include/model.php

```diff
@@ -550,7 +550,10 @@ public function count_posts( $lang, $q = array() ) {
 			$groupby = ' GROUP BY pll_tr.term_taxonomy_id';
 
 			if ( ! empty( $q['m'] ) ) {
-				$q['m'] = '' . preg_replace( '|[^0-9]|', '', $q['m'] );
+				$date_query = preg_replace( '|[^0-9]|', '', $q['m'] );
+				if ( is_string( $date_query ) ) {
+					$q['m'] = $date_query;
+				}
```

Comment:
I don't reproduce any error here on my local install. Do you?

Reply:
After updating my PHPStan dependency locally, I didn't reproduce the error...

Reply:
Revert in bf268a9